hscontrol/policy/v2/types.go

```diff
@@ -409,13 +414,44 @@ func (ag *AutoGroup) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-func (ag AutoGroup) Resolve(_ *Policy, _ types.Users, _ types.Nodes) (*netipx.IPSet, error) {
+func (ag AutoGroup) Resolve(p *Policy, users types.Users, nodes types.Nodes) (*netipx.IPSet, error) {
+	var build netipx.IPSetBuilder
+
 	switch ag {
 	case AutoGroupInternet:
 		return util.TheInternet(), nil
-	}
 
-	return nil, nil
+	case AutoGroupMember:
+		// autogroup:member represents all untagged devices in the tailnet.
+		for _, node := range nodes {
+			if len(node.ForcedTags) != 0 {
+				continue
+			}
+			if node.Hostinfo != nil && len(node.Hostinfo.RequestTags) != 0 {
+				continue
+			}
```

Comment:
See the comment bellow, this isnt really the ideal signal, because if the node requests a node, but it isnt allowed it, it still belongs to a user.

We should ultimately have `node.IsTagged()`, but that would be work for the next PR. 

This has been helpful because I've added things I didnt realise yet to #2417